indexer/services/comlink/src/controllers/api/v4/compliance-v2-controller.ts

```diff
@@ -276,6 +276,19 @@ router.post(
         complianceStatus,
         updatedAt,
       );
+      if (complianceStatus.length === 0 ||
+        complianceStatus[0] !== complianceStatusFromDatabase) {
+        if (complianceStatusFromDatabase !== undefined &&
+          complianceStatusFromDatabase.status !== ComplianceStatus.COMPLIANT
```

Comment:
when would it ever change from non-compliant to compliant?

Reply:
we're tracking states which are not COMPLIANT... there is a state where address has not status and becomes compliant which doesnt need to be tracked